app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp

```diff
@@ -0,0 +1,54 @@
+<%@ page pageEncoding="UTF-8" contentType="text/html;charset=UTF-8" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+
+<div class="row">
+    <div class="12 columns">
+        <h2>Download</h2>
+
+        <h3>Via FTP</h3>
+        <p>
+            You can download data for every dataset individually in Expression Atlas through our
+            <a href="http://ftp.ebi.ac.uk/pub/databases/microarray/data/atlas/experiments/" target="_blank">FTP site</a>.
+        </p>
+
+        <h3>From experiment pages</h3>
+        <p>
+            You can download single experiment files from the <i>Downloads</i> tab in any experiment page. Below is
+            a short summary about data that you can download.
+        </p>
+
+        <h4>Metadata Files</h4>
+        <p>
+            <img src="/gxa/sc/resources/images/download/download-tsv-file.png" alt="idf and sdrf file" align="bottom" class="icon"/>
+            Experiment Metadata: SDRF and IDF files containing information about the experiment.
+        </p>
+        <p>
+            <img src="/gxa/sc/resources/images/download/download-experiment-design.png" alt="experiment design file" align="bottom" class="icon"/>
+            Experiment Design file: Information about assays within the experiment.
+        </p>
+
+        <h4>Results Files</h4>
```

Comment:
We also added some new files, summary pdf https://www.pivotaltracker.com/story/show/179434417 and Unprocessed protein groups file  https://www.pivotaltracker.com/story/show/181094306. We also import some new icons with <i> tag or images in the atlas-experiments component. Please double check the file icons.

Reply:
@lingyun1010 the pivotal stories you mentioned refer to the new files added to the bulk expression atlas. I do not see any new types of files added to the download section for the single-cell experiments to which this PR is referring. Can you please check and let me know if any files are added for single cell experiments?

Reply:
Yes, you are right. No new files added to SC as far as I know.